ravenframework/Optimizers/mutators/mutators.py

```diff
@@ -182,23 +194,6 @@ def inversionMutator(offSprings, distDict, **kwargs):
 
   return offSprings
 
-def locationsGenerator(offSprings,locs):
-  """
-  Methods designed to process the locations for the mutators. These locations can be either user specified or
-  randomly generated.
-  @ In, offSprings, xr.DataArray, children resulting from the crossover process
-  @ In, locs, list, the two locations of the genes to be swapped
-  @ Out, loc1, loc2, int, the two ordered processed locations required by the mutators
-  """
-  if locs == None:
-    locs = list(set(randomUtils.randomChoice(list(np.arange(offSprings.data.shape[1])),size=2,replace=False)))
-    loc1 = np.minimum(locs[0], locs[1])
-    loc2 = np.maximum(locs[0], locs[1])
-  else:
-    loc1 = np.minimum(locs[0], locs[1])
-    loc2 = np.maximum(locs[0], locs[1])
-  return loc1, loc2
```

Comment:
Why you revert the changes? I would suggest to use function instead of duplicate codes.